README.md

```diff
@@ -0,0 +1,10 @@
+# ua-fixer
+
+several user agent fixes for Firefox Android, like making it a phone even if Firefox thinks tablet.
```

Comment:
Complete list:
- tablet => phone
- in mobile mode, pretend to be Chrome for Google Search
- always present a desktop user agent to Imgur

Reply:
To be fair, I had no idea what this project even was about.

Reply:
I literally explained it in chat ._.

Reply:
Not detailed these three points, though.